frontend/javascripts/oxalis/geometries/materials/plane_material_factory.ts

```diff
@@ -232,6 +226,14 @@ class PlaneMaterialFactory {
       this.uniforms[`${layerName}_unrenderable`] = {
         value: 0,
       };
+      const layer = getLayerByName(Store.getState().dataset, dataLayer.name);
+
+      this.uniforms[`${layerName}_transform`] = {
+        value: invertAndTranspose(layer.transformMatrix || Identity4x4),
```

Comment:
I'm not 100% sure what you mean, but it might be correct :smile: I added a comment to the `invertAndTranspose` function which explains its existence. Copied here so that you don't need to look it up:

> // Transposition is often needed so that the matrix has the right format
// for matrix operations (e.g., on the GPU; but not for ThreeJS).
// Inversion is needed when the position of an "output voxel" (e.g., during
// rendering in the fragment shader) needs to be mapped to its original
// data position (i.e., how it's stored without the transformation).
// Without the inversion, the matrix maps from stored position to the position
// where it should be rendered.